pkg/scheduler/plugins/priority/priority.go

```diff
@@ -112,6 +112,25 @@ func (pp *priorityPlugin) OnSessionOpen(ssn *framework.Session) {
 		return victims, util.Permit
 	}
 	ssn.AddPreemptableFn(pp.Name(), preemptableFn)
+	reclaimableFn := func(reclaimor *api.TaskInfo, reclaimees []*api.TaskInfo) ([]*api.TaskInfo, int) {
+		reclaimorJob := ssn.Jobs[reclaimor.Job]
+		var victims []*api.TaskInfo
+		for _, reclaimee := range reclaimees {
+			reclaimeeJob := ssn.Jobs[reclaimee.Job]
+			if reclaimeeJob == nil {
+				continue
+			}
+			if reclaimorJob.Priority > reclaimeeJob.Priority {
```

Comment:
why double `e`?

Reply:
I think it is like "Employee" vs "Employer"， so does "preemptor" vs "preemptee"